src/components/Match/renderMatch.js

```diff
@@ -56,7 +56,7 @@ function generateGraphData(match) {
   return {};
 }
 
-function generateTeamfights(match) {
+function generateTeamfights({ players, teamfights = [] }) {
```

Comment:
does this fail if teamfights is null?  The destructure only assigns a default value if the field is undefined?